packages/analytics/analytics-chart/src/composables/useExploreResultToDatasets.ts

```diff
@@ -99,7 +99,18 @@ export default function useExploreResultToDatasets(
             return [label, value]
           }))
 
-        const sortedDatasetIds = Object.entries(pivotRecords).sort(([, a], [, b]) => Number(b) - Number(a)).map(([key]) => (key.split(',')[0]))
+        const aggregatedPivotRecords = Object.keys(pivotRecords).reduce((acc, key) => {
```

Comment:
Would it be possible to add a unit test to `useExploreResultToDatasets` that checks this case?  I see it impacted the component test, but I think it'd be helpful to have something more explicit.